cookbook/integration_openai_sdk.ipynb

```diff
@@ -14,14 +14,18 @@
     },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    from langfuse.decorators import langfuse_context, observe
</pre><p>i'd include the import statement again just for clarity</p><p>from langfuse.openai import openai</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/langfuse/langfuse-docs/pull/415/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>